ruby/frontend/Dockerfile

```diff
@@ -6,4 +6,4 @@ RUN bundle install
 COPY frontend.ru /myapp
 
 EXPOSE 7000
-CMD [ "rackup", "frontend.ru", "--server", "puma", "--host", "0.0.0.0"]
+CMD [ "bundle", "exec", "rackup", "frontend.ru", "--server", "puma", "--host", "0.0.0.0"]
```

Comment:
Oh, I wondered about this in my recent Dockerfile. I couldn't recall if the `rackup` binary would just be whatever Bundler installed, thus making `bundle exec` unnecessary to execute the command. My image worked, so I guess that's the case...?

But I suppose one side effect is that `bundle exec` requires the Gemfile gems, which means you wouldn't need to `require` things manually in `frontend.ru`, which might be something worth going for.

Reply:
It _should_ work without `bundle exec`. This might be cruft left from when I was doing some shenanigans with pulling in an alternative version of OpenTelemetry Ruby checked out to the `vendor/` subdirectory. To have that directory included in the Ruby load path, I had to `bundle exec`.